proposals/001-ClusterRepresentation.md

```diff
@@ -0,0 +1,305 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
+# Cluster Representation in the Model
+
+This proposal introduces the concept of _cluster_ into the model.
+
+A _virtual cluster_ is a kafka cluster that the clients connect to.   The virtual cluster can be thought of as a 'virtual' cluster or a 'facade' to
```

Comment:
```suggestion
A _virtual cluster_ is a Kafka cluster that the clients connect to.  It can be thought of as a 'logical' cluster or a 'facade' to
```

Thinking either use the word logical here or just remove this part and make it `can be thought of as a facade..` .